stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/database_logic.py

```diff
@@ -1922,7 +1957,7 @@ async def execute_discovery_search(
         try:
             es_response = await search_task
         except exceptions.NotFoundError:
-            raise NotFoundError(f"Catalogs '{catalog_ids}' do not exist")
+            raise NotFoundError("Catalog or Collection missing during search.")
```

Comment:
Is it worth including which Catalog(s)/Collection(s) haven't been found?

Reply:
As a matter of fact, I don't think this error will ever be triggered here as the indices we are searching are only prefixes which are created at boot-time, therefore they will never be missing. I will update this to remove the unnecessary try-except block.